src/m365/aad/commands/o365group/o365group-user-remove.ts

```diff
@@ -42,47 +42,56 @@ class AadO365GroupUserRemoveCommand extends GraphCommand {
   }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
-    let userId = '';
     const groupId: string = (typeof args.options.groupId !== 'undefined') ? args.options.groupId : args.options.teamId as string;
 
-    const removeUser: () => void = (): void => {
-      const requestOptions: any = {
-        url: `${this.resource}/v1.0/users/${encodeURIComponent(args.options.userName)}/id`,
-        headers: {
-          accept: 'application/json;odata.metadata=none'
-        },
-        responseType: 'json'
-      };
-
-      request
-        .get<{ value: string; }>(requestOptions)
-        .then((res: { value: string; }): Promise<any> => {
-          userId = res.value;
-
-          const requestOptions: any = {
-            url: `${this.resource}/v1.0/groups/${groupId}/owners?$select=id,displayName,userPrincipalName,userType`,
+    const removeUser: () => void = async (): Promise<void> => {
+      try {
+        // retrieve userId
+        const userId: any = await request.get({
```

Comment:
Let's not use `any` and instead use the type of response to avoid refactoring issues in the future

Reply:
Implemented. Please also check [this](https://github.com/pnp/cli-microsoft365/discussions/2427) thread